joomla/joomla-cms
# [4.1][NO CACHE] Remove the math.div deprecation

Pull Request for Issue # .

Kinda replaces #36906 (no FA upgrade to v6)

### Summary of Changes
- patch the font awesome files
- upgrade to choices v10
- local fixes


### Testing Instructions
- Apply this Pr's branch
- run npm ci
- Check that nothing is broken visually and there are no console deprecation when running any node (css related) cli command

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

@brianteeman @bembelimen
